### PR TITLE
feat: send message twice enter or shift + enter

### PR DESCRIPTION
**Pull request by noahlias**
solve the event of send message

### CODE REVIEW

**Comment by p0lygun** [comment]
why is the input field so big ?
image

and send button removed ?

**Comment by noahlias** [comment]
sorry, it seems good on my mac. I need to  use my windows laptop  to fix the error
image

**Review by p0lygun — CHANGES_REQUESTED** [review]
https://github.com/Gorgeous-Ghouls/Blak/blob/a10c6970efa72696970624932be5631f1ca12ef8/client/app/ui/kv_files/chat_message.kv#L20-L24
adding `adaptive_height: True` should fix it

**Comment by p0lygun** [comment]
and why is send button removed ?

**Comment by noahlias** [comment]
It is like the modern chat app. and the screen in Figma didn't have the send button
image

**Comment by p0lygun** [comment]
ok 👍

**Review by p0lygun — DISMISSED** [review]

File: client/app/ui/kv_files/chat_message.kv [hunk 1]
```diff
@@ -13,14 +13,14 @@
...
-            #:set padding "10dp"  # you can set variable in kvlang
-            padding: padding, 0, padding, 0
             adaptive_height: True
             rows: 1
             MDBoxLayout:
+                adaptive_height: True
                 md_bg_color: Colors.accent_bg
                 radius: [5, 5, 0, 0 ]
                 padding: [5,0,0,5]
+                adaptive_height: True
```
Comment (p0lygun) [inline client/app/ui/kv_files/chat_message.kv]:
```suggestion
```
well now its there twice :rofl:

**Review by p0lygun — DISMISSED** [review]
just one last change

**Review by p0lygun — APPROVED** [review]
good job :partying_face:

**Pull request merged**